AirCasting/SearchAndFollow/CompleteScreen/Views/SearchAndFollowChartViewModel.swift

```diff
@@ -5,24 +5,29 @@ import Foundation
 import SwiftUI
 
 class SearchAndFollowChartViewModel: ObservableObject {
-    struct ChartDot {
-        let value: Double
-        let color: Color
-    }
+    
     struct ChartMeasurement {
         let value: Double
         let time: Date
     }
     
+    struct ChartDot {
+        let value: Double
+        let color: Color
+    }
+    
     @Published var entries: [ChartDot] = []
     
     let numberOfEntries = 9
     
-    func generateEntries(with measurements: [ChartMeasurement], thresholds: ThresholdsValue) -> (Date?, Date?) {
+    func generateEntries(with measurements: [ChartMeasurement], thresholds: ThresholdsValue, basedOn sensor: ChartSensor) -> (Date?, Date?) {
```

Comment:
`basedOn` -> `using` ?

Maybe also make it optional, so the filter is not required?

Reply:
I think that it is a lot of work needed to make it done. For now, when we are not filtering we are just going to return those measurements that we have passed. I would like to leave it at least for now as the time goes by. What would u say?